.gitignore

```diff
@@ -69,7 +69,6 @@ yarn-error.log
 
 #-----------------------------------------#
 
-/demo/
```

Comment:
I am not sure why the demo folder was ignored, seems unnecessary, as individual files can be defined in the `/demo/.gitignore` file as needed